### ISSUE TITLE
Estimated growth advantage over time

### ISSUE THREAD

**Issue by chaoran-chen** [GenSpectrum/cov-spectrum-website#580]
The estimates for the relative growth advantage change with new data. We would like to know how constant/accurate the initial estimates (and the CIs) are and how the estimates change over time.

Thus, we'd like to add a plot that shows the estimates and the CIs over time. On the x-axis, we'd have the date (**submission date or sampling date?**), and on the y-axis the estimated relative growth advantage.

How can we calculate this efficiently? The current relative advantage calculation is too slow.

**Comment by corneliusroemer** [comment]
Do I understand correctly that we'd like to better understand the biases of early estimates of growth advantages? To maybe be able to "nowcast" a "corrected" growth advantage?

**Comment by Sinickle** [comment]
@corneliusroemer Right, part of the reason is that I suspect that being able to see the growth advantage early time will help us understand how confident we can be in the current growth estimate -- if it is still fluctuating a lot from week to week, it probably isn't believable yet. I'm not sure if it would really be possible to nowcast it? I'm just hoping for something to make it more apparent how believable the current numbers are. 
Then, the second purpose -- I believe that if a previously stable growth advantage starts to change, it can indicate the effect of some genetic change that might be having an impact.
